kotlin-android/app/src/main/java/com/mongodb/tasktracker/TaskTracker.kt

```diff
@@ -14,6 +14,8 @@ lateinit var taskApp: App
 // global Kotlin extension that resolves to the short version
 // of the name of the current class. Used for labelling logs.
 inline fun <reified T> T.TAG(): String = T::class.java.simpleName
+const val PARTITION_EXTRA_KEY = "PARTITION"
+const val PROJECT_NAME_EXTRA_KEY = "PROJECT NAME"
```

Comment:
Unfamiliar with android programming so I'm a bit confused - are these hardcoded partition/project name values or are they only used by android for logging purposes?

Reply:
Basically when you launch a new activity in android you store your arguments for that activity in a big blob called an Intent and then look them up by a key. I define the key once and refer to that one definition when I store/access Intent values instead of maintaining the value in two places.

It's one of those very icky things in Android.